projects/plugins/jetpack/extensions/shared/jetpack-plugin-sidebar.js

```diff
@@ -16,6 +17,19 @@ const { Fill, Slot } = createSlotFill( 'JetpackPluginSidebar' );
 
 export { Fill as default };
 
+/**
+ * Open Jetpack plugin sidebar by default when URL includes query param.
+ */
+function openJetpackSidebar() {
+	dispatch( 'core/interface' ).enableComplementaryArea(
+		'core/edit-post',
+		'jetpack-sidebar/jetpack'
+	);
+}
+const queryParams = new URLSearchParams( window.location.search );
+const jetpackSidebarIsOpen = queryParams.get( 'jetpackSidebarIsOpen' );
```

Comment:
I wonder whether we should use the @wordpress/url package for this. Let me create suggestions set.